### DISCUSSION TITLE
TV 8,5

### DISCUSSION THREAD

**Discussion by yagiz-dev** [iptv-org/community#861] [Q&A]
### Channel ID

TV85.tr

### Notes

Official live stream: https://www.tv8bucuk.com/tv8-5-canli-yayin

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by BellezaEmporium** [comment]
Tokened, and it's using the ercdn lists.

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/blob/aec1bdb0a7037bf6f7112d2be3bdd93a50d5209f/streams/tr.m3u#L680-L681
